Mapbox/MapboxMapsStyleTests/ColorTests.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+#if canImport(MapboxMaps)
+@testable import MapboxMaps
+#else
+@testable import MapboxMapsStyle
+#endif
+
+internal class ColorTests: XCTestCase {
+
+    func testColorEncodingAndDecoding() throws {
+        let testColor = UIColor(red: 0.537, green: 0.812, blue: 0.941, alpha: 0.3)
+        encodeAndDecodeColor(testColor: testColor)
+    }
+
+    func testColorEncodingAndDecodingBlack() throws {
+        encodeAndDecodeColor(testColor: .black)
+    }
+
+    func testColorEncodingAndDecodingWhite() throws {
+        encodeAndDecodeColor(testColor: .white)
+    }
+
+    func testNonSRGBColor() throws {
+        let testColor = UIColor(patternImage: UIImage())
+        let color = ColorRepresentable(color: testColor)
+        XCTAssertNil(color)
+    }
+
+    func encodeAndDecodeColor(testColor: UIColor) {
```

Comment:
```suggestion
    func encodeAndDecodeColor(testColor: UIColor, line: UInt = #line) {
```

Reply:
And then forward the line parameter to the XCT* test assertions in this method. This will make it so that if one of them fails, Xcode will highlight the call site of this method which helps to identify which test failed.

Reply:
Nice!